src/student/StudentView.js

```diff
@@ -1,22 +1,61 @@
-import React from 'react';
-import { Alert, Container } from 'reactstrap';
-import Logo from '../logo.svg';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'reactstrap';
+import MainView from '../components/MainView';
 
-const StudentView = () => (
-  <div className="App">
-    <header className="App-header">
-      <img src={Logo} className="App-logo" alt="Logo" />
-      <h1 className="App-title">Welcome to the Graasp App Starter Kit</h1>
-    </header>
-    <Container className="App-body">
-      <Alert color="info">
-        This is the student view. Switch to the teacher view by clicking on the URL below.
-        <a href="?mode=teacher">
-          <pre>{`${window.location.host}/?mode=teacher`}</pre>
-        </a>
-      </Alert>
-    </Container>
-  </div>
-);
+class StudentView extends Component {
+  static propTypes = {
+    i18n: PropTypes.shape({}).isRequired,
+    t: PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      addClass: false,
+      intensity: 'moyen',
+    };
+    this.handleSection = this.handleSection.bind(this);
```

Comment:
You don't need to `bind` the handle to `this` if you are using `() => {}` functions, like you are using below.

Reply:
Got it.